docs/content/release/0.21.0-notes.rst

```diff
@@ -0,0 +1,274 @@
+.. _0.21.0_notes:
+
+===========================
+SimPEG 0.21.0 Release Notes
+===========================
+
+April 8th, 2024
+
+.. contents:: Highlights
+    :depth: 3
+
+Updates
```

Comment:
I believe that the SP simulation is also new in this release, is that right?

Reply:
Ah, nevermind, it looks like this was in 0.20
https://github.com/simpeg/simpeg/pull/1211